app/src/main/java/org/mozilla/fenix/settings/AccountSettingsFragment.kt

```diff
@@ -57,6 +57,8 @@ class AccountSettingsFragment : PreferenceFragmentCompat(), CoroutineScope {
         override fun onLoggedOut() {
             launch {
                 Navigation.findNavController(view!!).popBackStack()
+                val deviceNameKey = context!!.getPreferenceKey(R.string.pref_key_sync_device_name)
```

Comment:
The code here seems a bit brittle. Can onLoggedOut() occur when we're unattached? We pop the back stack, crashing if the view is null, and then ask for the context of the Fragment we just popped, crashing if it's null.

Reply:
Fixed!